float64 P
float64 D
float64 targetPos
float64 currentPos
float64 targetVel
float64 currentVel
float64 output
